app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextModuleActions.java

```diff
@@ -489,4 +509,190 @@ public void onClick(DialogInterface dialog, int id) {
                 });
         builder.show();
     }
+
```

Comment:
I think this would be worth to place in a seperate class, as this is quite a big amount now of TextActions :D. It's also planned to integrate this picker into all formats (so you also see this date-option e.g. in Plaintext format). So I think it would fit best in `format -> general -> something.java`. 
What do you think?

As of callbacks etc, you can use `Callback.a1<String>` for example, it's very easy fitting in one line :D!